docs/1.guide/5.routing.md

```diff
@@ -0,0 +1,108 @@
+---
+icon: solar:route-linear
+---
+
+## Registering Routes
+
+H3 app instance has a built in router system to register routes matching patterns with parameter or specific HTTP method.
+
+::tip
+Router is powered by [🌳 rou3](https://github.com/unjs/rou3), an ultra fast and tiny route matcher engine.
+::
+
+In the example below, we register a route requests to the `/hello` endpoint with HTTP GET method:
```

Comment:
The part "we register a route requests to ..." would need some more words for collocation (between "route" and "requests"), I guess.